.env.example

```
@@ -1,25 +1,63 @@
 # files, you should not have to change them. If you do, please do NOT commit
 # those changes
 NEXT_PUBLIC_VERCEL_ENV = development
 NEXT_PUBLIC_URL = 'http://localhost:3000'
 NEXT_PUBLIC_MOD_PROTOCOL_API_URL = 'https://api.modprotocol.org/api'
 
+NEXT_PUBLIC_ONCHAINKIT_PROJECT_NAME = 'Nounspace'
+NEXT_PUBLIC_ONCHAINKIT_API_KEY = ''
+
+# Optional (appearance and metadata)
+NEXT_PUBLIC_APP_ICON=https://your-app.vercel.app/icon.png
+NEXT_PUBLIC_APP_SUBTITLE = Short subtitle
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)